$OpenBSD: patch-src_server_h,v 1.1 2021/10/16 14:43:12 thfr Exp $

Remove dangerous localcommand function even if it it not turned on
by default and needs a command-line option. There is a pull request
at: https://github.com/deurk/mvdsv/pull/69


Index: src/server.h
--- src/server.h.orig
+++ src/server.h
@@ -1087,7 +1087,6 @@ void SV_SetClientConnectionTime(client_t* client);
 #ifdef SERVERONLY
 // mvdsv not changed over to enums yet, which was more about documentation
 #define SV_CommandLineEnableCheats() (COM_CheckParm("-cheats"))
-#define SV_CommandLineEnableLocalCommand() (COM_CheckParm("-enablelocalcommand"))
 #define SV_CommandLineDemoCacheArgument() (COM_CheckParm("-democache"))
 #define SV_CommandLineProgTypeArgument() (COM_CheckParm("-progtype"))
 #define SV_CommandLineUseMinimumMemory() (COM_CheckParm("-minmemory"))
@@ -1095,7 +1094,6 @@ void SV_SetClientConnectionTime(client_t* client);
 #define SV_CommandLineHeapSizeMemoryMB() (COM_CheckParm("-mem"))
 #else
 #define SV_CommandLineEnableCheats() (COM_CheckParm(cmdline_param_server_enablecheats))
-#define SV_CommandLineEnableLocalCommand() (COM_CheckParm(cmdline_param_server_enablelocalcommand))
 #define SV_CommandLineDemoCacheArgument() (COM_CheckParm(cmdline_param_server_democache_kb))
 #define SV_CommandLineProgTypeArgument() (COM_CheckParm(cmdline_param_server_progtype))
 #define SV_CommandLineUseMinimumMemory() (COM_CheckParm(cmdline_param_host_memory_minimum))
